### PR TITLE
Fix collectible issues

### PR DESCRIPTION
**Pull request by elgatovital**
Handles #886 
Handles #889
Handles #894 

Changes proposed in this pull request:
- Make shadow on top of collectible list item more visible
- Copy address to clipboard on click for LabeledAddressItem
- Hide view on token link on collectible details page

@gnosis/mobile-devs

### CODE REVIEW

File: app/src/main/res/layout/item_collectible_collectible.xml [hunk 1]
```diff
@@ -12,6 +12,7 @@
         android:layout_width="match_parent"
```
Comment (rmeissner) [inline app/src/main/res/layout/item_collectible_collectible.xml:12]: Why is it required to wrap this card view in a constraint layout?

File: app/src/main/res/layout/item_collectible_collectible.xml [hunk 2]
```diff
@@ -12,6 +12,7 @@
...
         android:layout_height="wrap_content"
         android:layout_gravity="center"
+        android:layout_marginTop="2dp"
```
Comment (rmeissner) [inline app/src/main/res/layout/item_collectible_collectible.xml:15]: This only increases the distance between the items, right? It does not add any more shadow.

Comment (elgatovital) [reply app/src/main/res/layout/item_collectible_collectible.xml:15]: top shadow/line was clipped by view bounds. We can fix it by either adding padding to parent or having this margin

File: app/src/main/res/layout/item_collectible_collectible.xml (hunk 1 above)
Comment (elgatovital) [reply app/src/main/res/layout/item_collectible_collectible.xml:12]: we want to have a white background for the whole width of the item

**Review by biafra23 — APPROVED** [review]

**Pull request merged**